src/main.cc

```diff
@@ -230,20 +231,35 @@ static int cli_rollback(LiteClient& client, const bpo::variables_map& params) {
   return static_cast<int>(aklite::cli::Rollback(akclient, src_dir.empty() ? nullptr : &local_update_source));
 }
 
-// clang-format off
-static const std::unordered_map<std::string, int (*)(LiteClient&, const bpo::variables_map&)> commands = {
-    {"daemon", daemon_main},
-    {"update", cli_update},
-    {"pull", cli_pull},
-    {"install", cli_install},
-    {"list", cli_list},
-    {"check", cli_check},
-    {"status", status_main},
-    {"finalize", cli_complete_install},
-    {"run", cli_complete_install},
-    {"rollback", cli_rollback},
+struct Cmd {
+  const std::string name;
+  const std::string description;
+  int (*func)(LiteClient&, const bpo::variables_map&);
 };
+
+// clang-format off
+static const std::array<Cmd, 10> commands = {{
+    {"daemon", "Start the update agent daemon", daemon_main},
+    {"update", "Update TUF metadata, download and install the selected target", cli_update},
+    {"pull", "Download or copy the target data to the device, to allow a install operation to be performed", cli_pull},
```

Comment:
I would remove `or copy`, if a offline bundle data are located on a thumb drive it is still download.

Reply:
Maybe just `Download the selected or specified target data`?

Reply:
Updated.